utils/utils.py

```diff
@@ -80,6 +81,12 @@ def remove_file(file_name):
     except OSError:
         pass
 
+def rmdir(dir_to_remove):
+    try:
+        shutil.rmtree(dir_to_remove)
+    except OSError:
+        pass
```

Comment:
Assuming OSError gets thrown when directory does not exist? Possibly better to just say if dir does not exist, pass? Not the biggest fan of relying on try/except for logic

Reply:
Yeah that's right. For this one, I don't mind since it is not a frequently executed command